photorepoapp-api/src/main/java/api/photorepoapp/photorepoappapi/models/Image.java

```diff
@@ -0,0 +1,94 @@
+package api.photorepoapp.photorepoappapi.models;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+
+import javax.persistence.*;
+
+import org.springframework.data.annotation.Id;
+
+public class Image {
+	
+	@Id
+	@GeneratedValue
+	private UUID id;
+	
+	private String name; 
+	
+	@GeneratedValue
+	private UUID userId;
```

Comment:
For now, let's not worry about User Ids, let's just focus on anonymously uploading files -- we can later incorporate a table for Users if we want to do authentication for login/registration.